packages/client/hmi-client/src/components/facets/numerical-facet.vue

```diff
@@ -157,7 +157,9 @@ export default defineComponent({
 });
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
+@import '@/styles/variables.scss';
```

Comment:
You can probably remove this now since you don't seem to be using this file? Or if this is needed for `facets-overrides.scss` you can import that file instead.

Reply:
Same as above